sdk/identity/identity/TOKEN_CACHING.md

```diff
@@ -68,6 +68,7 @@ The following table indicates the state of in-memory and persistent caching in e
 | `AuthorizationCodeCredential`  | Supported                                                              | Supported                     |
 | `AzureCliCredential`           | Not Supported                                                          | Not Supported                 |
 | `AzureDeveloperCliCredential`  | Not Supported                                                          | Not Supported                 |
+| `AzurePipelinesCredential`     | Supported                                                              | Supported                     |
```

Comment:
My assumption is that since `ClientAssertionCredential` supports both, `AzurePipelinesCredential` which uses `ClientAssertionCredential` under the hood does as well, but please do keep me honest considering that the SYSTEM_ACCESSTOKEN may differ between runs, etc.

Reply:
This is correct. I actually had a PR for the same.